### PR TITLE
Feature/2020 11 20 chaos engineering

### PR DESCRIPTION
**Pull request by Timverte**
### Thank you for your contribution! Please go through this small checklist to prepare your pull request to be reviewed as soon as possible :)
* [x] All authors were added to `_data/authors.yml` with a picture and an up-to-date bio
* [x] The blog post image is of ratio 500x293
  * Feel free to leave a comment if you don't know how to resize your image and someone will gladly help you out
* [x] The [**one sentence per line** method](https://raw.githubusercontent.com/brandon-rhodes/blog/master/texts/brandon/2012/one-sentence-per-line.rst) has been adopted
* [x] A spell check has been done
* [x] All links open in a new tab
  * Example: `[Amazing tech blog](https://ordina-jworks.github.io){:target="_blank" rel="noopener noreferrer"}`
* [x] The post's content properly resizes according to the window's size
  * Tip: Just shorten your window's horizontal length to easily test this
  * To have images properly resize use `class="image fit"`
  * To have YouTube videos properly resize, wrap the iframe into a `<div class="responsive-video">`
* [x] If a table of content is: check whether all links are working correctly

### Any specific remarks for the reviewers or anything you need help with?
\<enter text here>

### Checklist for the reviewers before merging
* [ ] Publishing date of the post has been correctly set
  * This can be done by renaming the filename or by setting the date in the header section of the post

### CODE REVIEW

**Comment by peterdekinder** [comment]
Twee opmerkingen:
* de table of contents is wel heel uitgebreid. Misschien een aantal chapters omvormen naar bold text of zoiets?
*  de externals links getiteld "Chaos Engineering post" linken beide naar hetzelfde artikel (via een redirect)

File: _posts/2020-11-20-chaos-engineering.md [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+* Race conditions
+* Downstream dependencies malfunction
+
+Real World Examples:
+
+At Netflix, they turn off machines because instance termination happens frequently in the wild and the act of turning off a server is cheap and easy.
+They simulate regional failures even though to do so is costly and complex,
+because a regional outage has a huge impact on their customers unless they are resilient to it.
+
+Or consider an organization that uses a messaging app such as Slack or HipChat to communicate during an incident.
```
Comment (KevinVHoutte) [inline _posts/2020-11-20-chaos-engineering.md:165]:
```suggestion
Consider an organization that uses a messaging app such as Slack or HipChat to communicate during an incident.
```

File: _posts/2020-11-20-chaos-engineering.md [hunk 2]
```diff
@@ -0,0 +1,500 @@
...
+Some good questions you can ask yourself to check the blast radius are:
+* How many customers are affected?
+* What functionality is impaired?
+* Which locations are impacted?
+
+Also try to have some sort of 'emergency button' you can use to cancel the experiment or to return to the normal state of the system in case you cannot find a solution.
+Be careful with experiments that modify the application state (cache or databases) or that can’t be rolled back easily or at all.
+
+Eventually when you start doing Chaos Experiments in your production environment,
+you will want to inform members of your organization about what you're doing, why you're doing it and when.
```
Comment (KevinVHoutte) [inline _posts/2020-11-20-chaos-engineering.md:198]:
```suggestion
you want to inform members of your organization about what you're doing, why you're doing it, and when.
```

File: _posts/2020-11-20-chaos-engineering.md [hunk 3]
```diff
@@ -0,0 +1,500 @@
...
+Or it can be as sophisticated as automatically designing and carrying out experiments in a production environment against a small but statistically significant fraction of live traffic.
+So, when starting out, it is a good practice to manually make your experiments and check the results,
+but running experiments manually is labor-intensive and ultimately unsustainable.
+So try to automate experiments and run them continuously.
+
+## Chaos GameDays & benefits
+
+Chaos GameDays are often known as days where a 'Master of Disaster' or a MoD, often in secret,
+will decide what kind of failure or disaster will happen on the system.
+He or She will generally start with something simple like the loss of capacity or the loss of connectivity.
```
Comment (KevinVHoutte) [inline _posts/2020-11-20-chaos-engineering.md:258]:
```suggestion
He or she will generally start with something simple like the loss of capacity or the loss of connectivity.
```

File: _posts/2020-11-20-chaos-engineering.md [hunk 4]
```diff
@@ -0,0 +1,500 @@
...
+Chaos Mesh is an open-source cloud-native Chaos Engineering platform that orchestrates chaos experiments in Kubernetes environments.
+It supports comprehensive types of failure simulation, including Pod failures, container failures,
+network failures, file system failures, system time failures, and kernel failures.  
+Chaos Mesh is one of the few open source tools to include a fully-featured web user interface (UI) called the Chaos Dashboard.
+However, its biggest limitations are its lack of node-level experiments,
+lack of native scheduling, and lack of time limits on ad-hoc experiments.
+
+### Litmus Chaos
+
+LitmusChaos Litmus is a toolset to do cloud-native chaos engineering.
```
Comment (KevinVHoutte) [inline _posts/2020-11-20-chaos-engineering.md:449]:
```suggestion
LitmusChaos is a toolset to do cloud-native chaos engineering.
```

File: _posts/2020-11-20-chaos-engineering.md [hunk 5]
```diff
@@ -0,0 +1,500 @@
+---
+layout: post
+authors: [tim_verte]
+title: 'Chaos Engineering'
+image: /img/2020-11-20-chaos-engineering/thumbnail.jpg
+tags: [Chaos Engineering, DevOps, Chaos, Backend, Netflix, Simian Army]
```
Comment (KevinVHoutte) [inline _posts/2020-11-20-chaos-engineering.md:6]: add resiliency

File: _posts/2020-11-20-chaos-engineering.md [hunk 6]
```diff
@@ -0,0 +1,500 @@
+---
+layout: post
+authors: [tim_verte]
+title: 'Chaos Engineering'
```
Comment (KevinVHoutte) [inline _posts/2020-11-20-chaos-engineering.md:4]:
```suggestion
title: 'Introduction into the world of Chaos Engineering'
```

**Review by KevinVHoutte — CHANGES_REQUESTED** [review]

**Review by KevinVHoutte — APPROVED** [review]

**Pull request merged**